Library/Homebrew/rubocops/extend/formula_cop.rb

```diff
@@ -0,0 +1,67 @@
+module RuboCop
+  module Cop
+    module Homebrew
+      class FormulaCop < Cop
+        @registry = Cop.registry
```

Comment:
Ah! That's a hack. I hadn't found a solution for it yet. Without this line, there's an error ```undefined method `enlist' for nil:NilClass``` being thrown.